cluster/gce/addons/podsecuritypolicies/persistent-volume-binder.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   privileged: false
   volumes:
   - 'nfs'
-  - 'secret'   # Required for service account credentials.
```

Comment:
This is already correct-- "service account" is a noun, this is not talking about servicing (verb) accounts.